### ISSUE TITLE
[Bug] HELP HELP HELP

### ISSUE THREAD

**Issue by paisekamao** [coqui-ai/TTS#3836]
### Describe the bug

I M trying to use this repo for urdu language i have found some pretrained module on hugging face 
but i m unable to use 
i dont have any prior knowledge of python i m not familiar with tts 
can some one pls help???????

### To Reproduce

from TTS.api import TTS

# Initialize TTS with a VITS model
model_name = "facebook/mms-tts-urd-script_devanagari"
tts = TTS(model_name=model_name)

# Define your text input
text = "Kya kehna chahte ho?"

# Generate and save speech
tts.tts_to_file(text=text, file_path="output.wav")

print("Audio saved at: output.wav")

### Logs

```shell
Exception has occurred: ValueError
not enough values to unpack (expected 4, got 2)
  File "E:\PROJECTS\Quotes_urdu\t2.py", line 27, in <module>
    tts = TTS(model_name=model_name)
ValueError: not enough values to unpack (expected 4, got 2)
```

### Environment

```shell
using windows 10
python 3.10.6
TTS                          0.22.0
```

**Comment by eginhard** [comment]
You can load this model in Coqui with:
```python
tts = TTS("tts_models/urd-script_devanagari/fairseq/vits")
```
`_arabic` or `_latin` would also work instead of `_devanagari` depending on your input text.

There is also a bug that you might encounter when loading fairseq models, which has been fixed in [our Coqui fork](https://github.com/idiap/coqui-ai-TTS), which you can install with `pip install coqui-tts`. This repository is not updated anymore.

**Comment by paisekamao** [comment]
can u please define in detail?
as i tried to run 


from TTS.api import TTS

tts = TTS("tts_models/urd-script_devanagari/fairseq/vits")

# Define your text input
text = "Kya kehna chahte ho?"

# Generate and save speech
tts.tts_to_file(text=text, file_path="output.wav")

print("Audio saved at: output.wav")


and got error as 
Exception has occurred: UnboundLocalError
local variable 'dataset' referenced before assignment
  File "E:\PROJECTS\Quotes_urdu\t2.py", line 25, in <module>
    tts = TTS("tts_models/urd-script_devanagari/fairseq/vits")
UnboundLocalError: local variable 'dataset' referenced before assignment

**Comment by Mkkrishnasai** [comment]
issue fixed in the PR #3994

**Comment by rose07** [comment]
https://tts.byylook.com/ai/text-to-speech
